blog-posts/return-to-office-interview.md

```diff
@@ -0,0 +1,142 @@
+Unser Konzept bei der Einstellung von Mitarbeitern hat sich gerade in der Situation als absolut richtig erwiesen. Gerade weil wir durchweg Leute in der Firma haben, die eigenverantwortlich arbeiten, die über den Tellerrand hinaus schauen können, die mitdenken und aus eigenem Antrieb anderen helfen, haben wir die herausfordernde Situation der letzten beiden Jahr so gut bewältigen können.
+
+**Wie produktiv war die Arbeit vom Home-Office aus? War das für euch befriedigend?**
+
+**Eric:** Die Produktivität in Bezug auf die Arbeit war sicher sehr gut. Aber befriedigend? Für mich persönlich war das teilweise total unbefriedigend. Weil man bei all den Erfolgen, die wir als Firma feiern konnten nie die Gelegenheit hatten, den Kollegen mal einen echten High-Five zu geben und gebührend zu feiern (lacht).
+
+**Gholam:** Ich empfand die beiden Jahre auch erstmal sehr produktiv. Das liegt natürlich daran, dass der Anteil am produktiven Output im Verhältnis zur Arbeitszeit gestiegen ist. Einfach weil der Coffee-Talk häufiger weggefallen ist und man sich voll auf die Arbeit konzentrieren konnte.
+Ob das für den Einzelnen befriedigend war, kann man so eindeutig glaube ich nicht beantworten. Auf der einen Seite war die Gesamtsituation natürlich sehr belastend. Auf der anderen Seite hat sich auch die Perspektive auf die Arbeit, auf Work-Life-Balance, auf Remote-Arbeit zum Besseren geändert.
+
+**Eric:** Insgesamt würde ich die Frage ob “befriedigend” so auch nicht beantworten können. Die Jahre waren eben einfach herausfordernd. Man konnte sich nicht mehr auf die bewährten Prozesse verlassen, sondern musste neue definieren. Das ist uns aber recht gut gelungen.
+
+**Neben der Frage, wie produktiv wir arbeiten, gibt es ja auch andere Faktoren, die ausmachen, ob und wie wir als Team gut funktionieren. Wie habt ihr die Entwicklung der Firmenkultur unter diesen Voraussetzungen empfunden? Das war doch sicher auch eine ganz spezielle Herausforderung?**
+
+**Gholam:** In der aktuellen Teamzusammensetzung war unsere Vorstellung von Kultur schon so weit verinnerlicht, dass diese beim Wechsel ins Home-Office einfach erhalten und weitergelebt werden konnte. Dabei geht es mir vor allem darum wie wir als Team und als Kollegen miteinander umgegangen sind. Das hat auch remote sehr gut geklappt.
+Nur mal als Beispiel: Fabian hat sich bereit erklärt die Weihnachtsfeier zu planen. Jesús hat sich die Onboarding-Prozesse vorgenommen und optimiert. Das musste nicht von uns vorgegeben werden, sondern ist von den Mitarbeitern eigenverantwortlich vorangetrieben worden.
+
+<br />
+
+![Virtuelle Weihnachtsfeier @ Satellytes. 🙏 Georgios and Fabian für die Organisation!](images/return-to-office-interview-christmas-party.png)
+<small>Virtuelle Weihnachtsfeier @ Satellytes. 🙏 Georgios and Fabian für die Organisation!</small>
```

Comment:
for me it reads natural to directly translate the 🙏 emoji to "thanks", but since this is not a very informal chat, I'd consider actually adding the word "thanks" to the emoji